tests/cli/net_handler_cli.c

```diff
@@ -299,42 +292,51 @@ void prompt_device_data() {
                 return;
             }
 
+            // Assign type and make sure it's an integer
+            dev_type = (uint8_t) strtol(nextcmd, NULL, 10);
             errno = 0;
```

Comment:
`errno` needs to be set to 0 before you call `strtol`.

Reply:
You're right! That's a mistake--I'll make that change (with a comment explaining why).